CHANGELOG.md

```diff
@@ -1,10 +1,19 @@
-
 # Changelog
 All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.11.3] - 2023-02-23
+
+### Added
+ - Add 403 invalid token error response
```

Comment:
As a third party this won't be clear to me what has been changed. In my view, it is also a 'Changed' instead of an 'Added'.